src/Microsoft.CodeAnalysis.Analyzers/Core/AnalyzerReleases.Shipped.md

```diff
@@ -1,6 +1,7 @@
 ## Release 3.0.0
 
 ### New Rules
+
```

Comment:
You should also fix the code fixer that generates these tables, probably in a separate PR though. I am presuming the analyzer already ignores blank lines and hence is unaffected.

Reply:
@mavasani It was already fixed by @Evangelink.

https://github.com/dotnet/roslyn-analyzers/pull/4825